spring-boot-starters/langchain4j-community-xinference-spring-boot-starter/src/main/java/dev/langchain4j/community/xinference/spring/AutoConfig.java

```diff
@@ -0,0 +1,183 @@
+package dev.langchain4j.community.xinference.spring;
+
+import static dev.langchain4j.community.xinference.spring.Properties.PREFIX;
+
+import dev.langchain4j.community.model.xinference.XinferenceChatModel;
+import dev.langchain4j.community.model.xinference.XinferenceEmbeddingModel;
+import dev.langchain4j.community.model.xinference.XinferenceImageModel;
+import dev.langchain4j.community.model.xinference.XinferenceLanguageModel;
+import dev.langchain4j.community.model.xinference.XinferenceScoringModel;
+import dev.langchain4j.community.model.xinference.XinferenceStreamingChatModel;
+import dev.langchain4j.community.model.xinference.XinferenceStreamingLanguageModel;
+import org.springframework.boot.autoconfigure.AutoConfiguration;
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty;
+import org.springframework.boot.context.properties.EnableConfigurationProperties;
+import org.springframework.context.annotation.Bean;
+
+@AutoConfiguration
+@EnableConfigurationProperties(Properties.class)
+public class AutoConfig {
+    @Bean
+    @ConditionalOnProperty(PREFIX + ".chat-model.base-url")
+    public XinferenceChatModel chatModel(Properties properties) {
```

Comment:
```suggestion
    public XinferenceChatModel xInferenceChatModel(Properties properties) {
```

Using `chatModel` as bean name may make conflict with other bean.